apierr/errors.go

```diff
@@ -224,6 +224,7 @@ func parseUnknownError(resp *http.Response, requestBody, responseBody []byte, er
 
 func MakeUnexpectedError(resp *http.Response, err error, requestBody, responseBody []byte) error {
 	rts := httplog.RoundTripStringer{
+		Request:                  resp.Request,
```

Comment:
It might be good to add a check as Pieter mentioned it's exported, since we were using response in line 227 before without null check, it should be safe:
`Response: resp,`